src/pages/Shopping/shoppingList.js

```diff
@@ -1,37 +1,61 @@
 import React, { Component } from 'react';
-import Item from './Item/Item';
+import Category from './Category/Category';
+import Item from '../../components/Item/Item';
 
-import './shoppingList.scss';
+import './ShoppingList.scss';
+
+class ShoppingList extends Component {
+  state = {
+    items: [],
+    name: [
+      {
+        id: 1,
+        name: 'allmeat',
+      },
+      {
+        id: 2,
+        name: 'pig',
+      },
+
+      {
+        id: 3,
+        name: 'beef',
+      },
+      {
+        id: 4,
+        name: 'chicken',
+      },
+      {
+        id: 5,
+        name: 'susan',
+      },
+    ],
+  };
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/itemList.json', {
+      headers: {
+        method: 'GET',
+      },
+    })
+      .then(res => res.json())
+      .then(data => this.setState({ items: data }));
+  }
 
-export class shoppingList extends Component {
   render() {
     return (
       <section className="shoppingList">
         <div className="categoryImg">
-          <img
-            src="https://lh3.googleusercontent.com/proxy/XnO8ZKm0Rh7o8xnbW6jA-CC0BPGasCGJl_5Gpesc5Wa3AIjDAjmgeOj5g6I0fbHIHIedNJLIobN1XAAtRY2oQQKiclqLZuVtkFJuWzBun9OfhV3LhvViloBwA9oLJe-xSumO9NAisFYhXkr_5j5PIgGuTgauEYWOZ903cNOyjym8qXIsvJp5wLwlbITSV1WVUdW9pg"
-            alt="categoryImg"
-          />
+          {this.props.match.params.name ? (
+            <img
+              src={`/images/${this.props.match.params.name}.png`}
+              alt="categoryImg"
+            />
+          ) : (
+            <img src={`/images/allmeat.png`} alt="categoryImg" />
+          )}
```

Comment:
- this.props.match.params.name 많이 사용하시는데, 비구조화 할당하시면 좀 더 간편하지 않을까요?
- 달라지는 부분에 대해서만 삼항연산자를 사용해주세요!
```suggestion
            <img
              src={name ? `/images/${this.props.match.params.name}.png` : '/images/allmeat.png'}
              alt="categoryImg"
            />
```

Reply:
와우.. 요런방법이! 수정했습니다 감사합니다!👻